contrib/completions/bash/buildah

```diff
@@ -559,13 +565,19 @@ return 1
      -h
      --quiet
      -q
+     --tls-verify
+     --remove-signatures
     "
 
      local options_with_args="
      --chown
      --chmod
      --contextdir
      --ignorefile
+     --authfile
+     --cert-dir
+     --creds
+     --decryption-key
```

Comment:
ditto --from question

Reply:
Can you elaborate on this?  I don't understand.

Reply:
It's set now, I was asking to have the `--from` added here too.